src/main/resources/journals/journalList.csv

```diff
@@ -450,6 +450,7 @@ Advanced Materials;Adv. Mater.;;
 Advanced Materials and Processes;Adv. Mater. Processes;;
 Advanced Materials for Optics and Electronics;Adv. Mater. Opt. Electron.;;
 Advanced Materials Forum;Adv. Mater. Forum;;
+Advanced Modeling and Simulation in Engineering Sciences;Adv. Model. and Simul. in Eng. Sci.;;
```

Comment:
``and`` or ``in`` are left out in the ISO specs, so I would expect ``Adv. Model. Simul. Eng. Sci.``

Reply:
@jlaehne You're right. I will fix this.